cachito/web/migrations/versions/02229e089b24_delete_user_username_index.py

```diff
@@ -0,0 +1,26 @@
+"""Remove User.username unique index
+
+Revision ID: 02229e089b24
+Revises: 7d979987402d
+Create Date: 2021-12-15 21:37:30.582812
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "02229e089b24"
+down_revision = "7d979987402d"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    with op.batch_alter_table("user", schema=None) as batch_op:
+        batch_op.drop_index("ix_user_username")
+
+
+def downgrade():
```

Comment:
to downgrade it we will not be able to get all the user names, because they were deleted, right? I think we will need to get a backup before releasing it.

Reply:
The only thing that's being deleted here is the index, no changes will be made to the actual data :)